api/geth_backend.go

```diff
@@ -201,6 +201,24 @@ func (b *GethStatusBackend) GetAccounts() ([]multiaccounts.Account, error) {
 	return b.multiaccountsDB.GetAccounts()
 }
 
+func (b *GethStatusBackend) getAccountByKeyUID(keyUID string) (*multiaccounts.Account, error) {
+	b.mu.Lock()
+	defer b.mu.Unlock()
+	if b.multiaccountsDB == nil {
+		return nil, errors.New("accounts db wasn't initialized")
```

Comment:
Sorry, probably dumb question - when might it happen that `multiaccountsDB == nil`?

Reply:
Not dumb at all 🙂 It depends on if `OpenAccounts` get called correctly, but whatever, keeping it is better than missing it, when invoking function on a nil object would cause strange error stacktrace